panther_ioc_rules/log4j_exploit_iocs.py

```diff
@@ -0,0 +1,24 @@
+from panther_analysis_tool.immutable import ImmutableList
+from panther_iocs import LOG4J_EXPLOIT_IOCS
+
+
+def nested_value_search(data, search_value):
+    if not hasattr(data, "__iter__"):
+        return False
+    if isinstance(data, ImmutableList):
+        return any(nested_value_search(item, search_value) for item in data)
+
+    if isinstance(data, dict):
+        return any(nested_value_search(v, search_value) for v in data.values())
+
+    return search_value in data
+
+
+def rule(event):
+    event_dict = dict(event)
```

Comment:
you can also call `event.to_dict()` here, [ref](https://github.com/panther-labs/panther_analysis_tool/blob/master/panther_analysis_tool/immutable.py#L97)